components/common/src/main/java/com/hotels/styx/common/StyxFutures.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2013-2017 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.styx.common;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+import static java.lang.Thread.currentThread;
+
+/**
+ * An utility class to make your life easier with CompletableFutures.
+ */
+public final class StyxFutures {
+    private StyxFutures() {
+    }
+
+    /**
```

Comment:
Unfinished Javadoc.

Reply:
Not sure what you refer to?

Reply:
The Javadoc on that method does not have any text for its parameter, type parameter or return type. Your IntelliJ settings should highlight this.